deeppavlov/core/commands/utils.py

```diff
@@ -42,3 +42,13 @@ def get_deeppavlov_root() -> Path:
 
 def expand_path(path: Union[str, Path]) -> Path:
     return get_deeppavlov_root() / Path(path).expanduser()
+
+
+def is_empty(d: Path) -> bool:
+    """
+    Check if directory is empty.
+    """
+    for x in d.iterdir():
+        if x.is_file():
+            return True
+    return False
```

Comment:
Directory is empty when there's a file?

Reply:
Switch True and False :D

Reply:
And what if there's a directory, but no files?

Reply:
So you suggest what, recursive checking?

Reply:
You could return in first iteration without checking anything. or just `bool(list(d.iterdir()))`